core-api/.vscode/launch.json

```diff
@@ -16,7 +16,7 @@
                 "--port", "5002"
             ],
             "jinja": true,
-            "envFile": "${workspaceFolder}/../.env"
+            "envFile": "${workspaceFolder}/../.env.local"
```

Comment:
It has to contain secrets, because the vscode python plugin doesn't support multiple env files, so we can't really. This is really messy and I don't like having a separate env file for this purpose but it is what it is unfortunately